e2e/utils.go

```diff
@@ -877,3 +898,10 @@ func checkDataPersist(pvcPath, appPath string, f *framework.Framework) error {
 	err = deletePVCAndApp("", f, pvc, app)
 	return err
 }
+
+func handleFlags() {
```

Comment:
can you move this to https://github.com/ceph/ceph-csi/blob/master/e2e/e2e_test.go